src/Bridges/Constraint/det.jl

```diff
@@ -99,7 +99,7 @@ end
 
 MOI.supports_constraint(::Type{LogDetBridge{T}}, ::Type{<:Union{MOI.VectorOfVariables, MOI.VectorAffineFunction{T}}}, ::Type{MOI.LogDetConeTriangle}) where T = true
 MOIB.added_constrained_variable_types(::Type{<:LogDetBridge}) = Tuple{DataType}[]
-MOIB.added_constraint_types(::Type{LogDetBridge{T}}, ::Type{<:Union{MOI.VectorOfVariables, MOI.VectorAffineFunction{T}}}, ::Type{MOI.LogDetConeTriangle}) where T = [(MOI.VectorAffineFunction{T}, MOI.PositiveSemidefiniteConeTriangle), (MOI.VectorAffineFunction{T}, MOI.ExponentialCone), (MOI.ScalarAffineFunction{T}, MOI.LessThan{T})]
+MOIB.added_constraint_types(::Type{LogDetBridge{T}}) where T = [(MOI.VectorAffineFunction{T}, MOI.PositiveSemidefiniteConeTriangle), (MOI.VectorAffineFunction{T}, MOI.ExponentialCone), (MOI.ScalarAffineFunction{T}, MOI.LessThan{T})]
```

Comment:
Is this an un-related bug fix? Do we not have tests?

Reply:
It's now required for `fill_...` to work for LazyBridgeOptimizer. I have made a independent PR with additional tests: https://github.com/JuliaOpt/MathOptInterface.jl/pull/870